scripts/artifacts-building/apt/apt-artifacts-testing.yml

```diff
@@ -0,0 +1,67 @@
+---
+- hosts: localhost
+  connection: local
+  gather_facts: no
+  vars:
+    package_files_path: /tmp/Packages-files
+  tasks:
+    - file:
+        path: "{{ package_files_path }}"
+        state: directory
+    - name: Fetch rpc-repo package files
+      get_url:
+        url: "http://rpc-repo.rackspace.com/apt-mirror/integrated/dists/{{ rpc_release }}-{{ item[0] }}/{{ item[1] }}/binary-{{ item[2] }}/Packages"
```

Comment:
This is pulling from the published package list. Surely it should test against the package list of the repo we just built?